### DISCUSSION TITLE
Menu Size

### DISCUSSION THREAD

**Discussion by EureDudeheit** [zim-desktop-wiki/zim-desktop-wiki#2501] [Q&A] [answered]
Hello,
i am using zim with Kubuntu 22.04. The menu bar as well the sidebars (index, tags) appear very small. Is there any way to change this?

**Comment by introt** [comment, marked as answer]
Hi,

Are you using a HiDPI screen? Since you're using a Qt-based desktop environment, you might need to set a different scaling or DPI (pixel density) setting for Gtk apps like Zim. You might find the "KDE Plasma" section of https://wiki.archlinux.org/title/HiDPI helpful.

**Reply by EureDudeheit** [reply]
Thank you, setting the scaling to a higher value did the trick.
